src/Symfony/Component/Form/Tests/Extension/Validator/DataTransformer/ViewObjectTransformer.php

```diff
@@ -0,0 +1,73 @@
+<?php
+namespace Symfony\Component\Form\Tests\Extension\Validator\DataTransformer;
+
+use Symfony\Component\Form\DataTransformerInterface;
+use Symfony\Component\Form\Exception\TransformationFailedException;
+use Symfony\Component\PropertyAccess\PropertyAccess;
+
+/**
+* Transforms Object to array for view
+*
+* @author Charles J. C. Elling <charles@denumeris.com>
+*/
+class ViewObjectTransformer implements DataTransformerInterface {
```

Comment:
What's the use case for this transformer? It seems to me that this is not needed since you are duplicating the logic of the [`PropertyPathMapper`](https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Form/Extension/Core/DataMapper/PropertyPathMapper.php#L24) which is used by default for both arrays and objects.